tests/components/portainer/test_binary_sensor.py

```diff
@@ -0,0 +1,31 @@
+"""Tests for the Portainer binary sensor platform."""
+
+from unittest.mock import AsyncMock, patch
+
+from syrupy import SnapshotAssertion
+
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry as er
+
+from . import setup_integration
+
+from tests.common import MockConfigEntry, snapshot_platform
+
+
+async def test_all_entities(
+    hass: HomeAssistant,
+    snapshot: SnapshotAssertion,
+    mock_portainer_client: AsyncMock,
```

Comment:
that variable is not accessed within the test, so we don't need the fixture as parameter, we can use `pytest.mark.usefixtures` 
```suggestion
@pytest.mark.usefixtures("mock_portainer_client")
async def test_all_entities(
    hass: HomeAssistant,
    snapshot: SnapshotAssertion,
```

Reply:
Isn't this more preference? I see tons of tests written this way and I've done so for quite a long time. ;)